CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Changelog
 Running changelog of releases since `1.0.0-beta01`
 
-## v1.0.0-beta03
+## v1.0.0
 
 ### Features
 
-- Added OktaContext.AuthenticationFailed event
+- Added OktaContext convenience methods and events
```

Comment:
Can you please add more details on what new methods and events were introduced in this version? Same for Okta state manager.